src/components/PlayerView/index.tsx

```diff
@@ -76,6 +77,17 @@ export function PlayerView({
   ...props
 }: PlayerViewProps) {
   // Native view reference.
+  const fakeStateUpdater = useState(1)[1];
```

Comment:
The comment above is no longer at the right position.. that said, I think we can remove it anyways.

Reply:
Done: d87a4452dc7eaccf426b4d696fb3f28889b19b29